File: src/musicgo/src/shared/classes/ContractViewModel.ts
import { ServiceViewModel } from './ServiceViewModel';
import { UserViewModel } from './UserViewModel';
import { Guid } from "guid-typescript";
export class ContractViewModel{
    public Id: string = '00000000-0000-0000-0000-000000000000';
    public HireDate : Date = new Date();
    public ExpirationDate : Date = new Date();
    public Service : ServiceViewModel | null = new ServiceViewModel();
    // public User : UserViewModel | null = new UserViewModel();

}

File: ViewModels/ViewModels/ContractViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ViewModels.ViewModels
{
    public class ContractViewModel : BaseViewModel
    {

        public DateTime HireDate { get; set; }

        public DateTime ExpirationDate { get; set; }

        public ServiceViewModel Service { get; set; }

        public UserViewModel User { get; set; }
    }
}
